src/components/CouponInput/CouponInput.js

```diff
@@ -177,7 +179,7 @@ class CouponInput extends Component {
             showMessage={showMessage && !suppressMessage}
             messageType={messageType}
           >
-            {message}
+            {t(message)}
```

Comment:
We are missing the key here + translation for successful coupon application is not working